crt_portal/cts_forms/views.py

```diff
@@ -226,6 +227,17 @@ def index_view(request):
             report.protected_class.all().order_by('form_order'),
             report.other_class,
         )
+        # If a user has an email, it is looked up in the table to see if they are a repeat writer.
+        if report.contact_email:
+            repeat_writer = RepeatWriterInfo.objects.filter(email=report.contact_email).first()
+            if repeat_writer:
+                report.related_reports_count = repeat_writer.email_count
+            # If the email is not in the repeat_writer table, add it and initialize to 1.
+            else:
+                report.related_reports_count = 1
```

Comment:
@billyhunt Why is the default value 1? There are no related reports if they aren't a repeat writer, so shouldn't it be 0?

Reply:
Okey dokey. Yeah that makes the most sense. Keep it consistent and effectively related reports = total reports. Thanks!